assets/src/edit-story/components/canvas/layout.js

```diff
@@ -115,14 +115,14 @@ const PageAreaFullbleedContainer = styled(Area).attrs({
   display: flex;
   justify-content: center;
   align-items: center;
-  background-color: ${({ theme }) => theme.colors.fg.v1};
 `;
 
 const PageAreaSafeZone = styled.div`
   width: 100%;
   height: var(--page-height-px);
   overflow: visible;
-  position: relative;
+  position: absolute;
```

Comment:
If you still prefer `position: relative`, the `margin: auto 0` might work for you here. And it'd definitely work for `position:absolute` instead of `top`. Not sure if that's more useful in your case however. PTAL.